pkg/backporter/backporter.go

```diff
@@ -162,22 +170,49 @@ const clonesTemplateConstructor = `
 			<tr> <td colspan=4 style="text-align:center;"> No clones found! </td></tr>
 		{{ end }}
 		</tbody>
-	</table>`
+	</table>
+	<form class="form-inline my-2 my-lg-0" role="search" action="/createclone" method="post">
+		<input type="hidden" name="ID" value="{{.Bug.ID}}">
+		<select class="form-control mr-sm-2" aria-label="Search" name="version" id="target_version" required>
+			<option value="" disabled selected hidden>Target Version</option>
+			{{ range $version := .CloneTargets }}
+				<option value="{{$version}}" id="opt_{{$version}}">{{$version}}</option>
+			{{end}}
+		</select>
+		<button class="btn btn-outline-success my-2 my-sm-0" type="submit">Create Clone</button>
+    </form>
+</div>`
 
+const errorTemplateConstructor := `
+<div class="alert alert-danger" id="error-banner">
+<a href="#" class="close" data-dismiss="alert" aria-label="close">&times;</a>
+<strong>Error! </strong> <label id ="error-text">{{.}}</label>
+</div>`
 var (
 	clonesTemplate = template.Must(template.New("clones").Parse(clonesTemplateConstructor))
 	emptyTemplate  = template.Must(template.New("empty").Parse("{{.}}"))
+	errorTemplate = template.Must(template.New("error").Parse(errorTemplateConstructor))
+	targetVersions = sets.NewString(
+		"4.7.0",
```

Comment:
This needs to be configurable as it will change over time. Why did you add the `3.0` and `2.0`?

Reply:
Those were test values - I havent finalized that part yet - I'll try to get them using flags like its being done somewhere upstream